source/basic.tex

```diff
@@ -4147,7 +4147,7 @@
 \begin{note}
 The runtime pointer alignment function\iref{ptr.align}
 can be used to obtain an aligned pointer within a buffer; the aligned-storage templates
-in the library\iref{meta.trans.other} can be used to obtain aligned storage.
+in the library\iref{depr.meta.types} can be used to obtain aligned storage.
```

Comment:
Instead of changing the cross-reference to properly reference the deprecated feature, should we suggest a mechanism that avoids the deprecated feature? Something like:
```tex
can be used to obtain an aligned pointer within a buffer; 
an \grammarterm{alignment-specifier}\iref{dcl.align} can be used
to align storage explicitly.
```

Reply:
@jensmaurer Could you please? That sounds like a better solution than to refer to [depr].

Reply:
@jensmaurer: friendly ping?

Reply:
pong

Reply:
Thanks!